.circleci/config.yml

```
@@ -259,15 +259,10 @@ jobs:
       tag: "16.18.0"
     steps:
       - checkout
-      - generate-checksums
       - install-node-modules
```

```suggestion
      - generate-checksums
      - install-node-modules
```
we need this for cache